### PR TITLE
[SMALLFIX] Remove the several redundant run commands

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Remove the several redundant run commands.

### Why are the changes needed?

Remove the several redundant run commands.

### Does this PR introduce any user facing changes?

NA

### CODE REVIEW

File: bin/alluxio-start.sh [hunk 1]
```diff
@@ -47,9 +44,6 @@ MOPT (Mount Option) is one of:
            specified directory (relative to each worker node's host filesystem).
 -f         format Journal, UnderFS Data and Workers Folder on master.
 -h         display this help.
--i backup  a journal backup to restore the master from. The backup should be
```
Comment (Xenorith) [inline bin/alluxio-start.sh:50]: if you remove the `-i` from usage description, it should also be removed from the opts parsing under `main() {` as well as any logic parts that it affects

Comment (yyongycy) [reply bin/alluxio-start.sh:50]: Good catch, updated

**Comment by Xenorith** [comment]
removed even more

**Review by Xenorith — APPROVED** [review]

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**